src/tests/banners/UsEoyAppealBannerTest.ts

```diff
@@ -0,0 +1,66 @@
+import { BannerPageTracking, BannerTargeting, BannerTest } from '../../types/BannerTypes';
+import { usEoyAppeal, usEoyAppealWithVisual } from '../../modules';
+import { TickerCountType, TickerEndType } from '../../lib/variants';
+
+const tickerSettings = {
+    countType: TickerCountType.money,
+    endType: TickerEndType.unlimited,
+    currencySymbol: '$',
+    copy: {
+        countLabel: 'contributions',
+        goalReachedPrimary: "We've hit our goal!",
+        goalReachedSecondary: 'but you can still support us',
+    },
+};
+
+const isLive = false;
+
+export const UsEoyAppealNonSupportersBanner: BannerTest = {
```

Comment:
both of these tests needs to define a `locations` field

Reply:
Ahhh ofc!